styles/style.css

```diff
@@ -7,4 +7,112 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-image: url("../images/bgtile.png");
+  background-repeat: repeat-x;
+}
+p {
+  font-size: 12px;
+}
+h2 {
+  font-size: 20px;
+  color: #06365f;
+}
+.box-one {
+  /*border: 1px solid red;*/
+  width: 100 vw;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+.box-two {
+  /* border: 2px solid yellow;*/
+  display: flex;
+  flex-direction: row;
+  padding-top: 100px;
+}
+.black-box {
+  /*border: 2px solid black;*/
+}
+.phones-img {
+  /*border: 2px solid blue;*/
+  width: 260px;
+  padding-top: 100px;
+  padding-left: 200px;
```

Comment:
this padding-left is changing the position of all your design. you can remove it to properly centralize the elements